Open/nuxt.config.ts

```diff
@@ -15,11 +15,27 @@ export default {
             // { name: "viewport", content: "width=device-width, initial-scale=1" }, TODO: Fix mobile
             { hid: "description", name: "description", content: "Corsace Open is one of the largest 4v4 tournaments in osu! Hailing great competition and entertaining matches." },
```

Comment:
The state's `tournament` has a `description` property fwiw so all the description fields can just be replaced with that